### ISSUE TITLE
app.detect_dependencies == true?

### ISSUE THREAD

**Issue by tkadauke** [clayallsopp/motion-require#8]
This might be more a question or an issue with RubyMotion, but I just ran into a stack overflow using motion_require. The actual problem probably lies deep within the RubyMotion build system. It went away once I set app.detect_dependencies = false. Maybe motion-require should set that to false by default?

**Comment by clayallsopp** [comment]
Yeah, I ran into this problem too (in general). The problem is another library required in someone's app can just override that and make it not work again, since `detect_dependencies` is just boolean for all files. Temporarily I just turn it off for all my stuff.

I submitted https://github.com/HipByte/RubyMotion/pull/81 as a clean workaround, but they don't have a comment on it yet. The idea is that all files using `motion_require` would be un-included from `detect_dependencies`, but everything else would work as expected. Maybe +1 that to bump it in their inbox?
